src/backoffice/settings/languages/workspace/language/language-workspace.context.ts

```diff
@@ -0,0 +1,72 @@
+import { UmbLanguageStore, UmbLanguageStoreItemType } from '../../language.store';
+import type { LanguageDetails } from '@umbraco-cms/models';
+import { UmbControllerHostInterface } from 'src/core/controller/controller-host.mixin';
```

Comment:
Please use alias paths instead and go through the files with similar paths.

```suggestion
import { UmbControllerHostInterface } from '@umbraco-cms/controller';
```